### PR TITLE
Fix /_status HARAKIRI timeout

### PR DESCRIPTION
**Pull request by fantix**
### Bug Fixes
* Align uWSGI timeout of `/_status` requests

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
Jenkins Build 1 : time taken 30 min
          Check the https://jenkins.planx-pla.net/job/perf-sheepdog/job/PR-262/1/display/redirect

**Comment by PlanXCyborg** [comment]
Jenkins Build 3 : time taken 40 min
          Check the https://jenkins.planx-pla.net/job/perf-sheepdog/job/PR-262/3/display/redirect

**Comment by PlanXCyborg** [comment]
Jenkins Build 4 : time taken 35 min
          Check the https://jenkins.planx-pla.net/job/perf-sheepdog/job/PR-262/4/display/redirect

**Comment by PlanXCyborg** [comment]
Jenkins Build 5 : time taken 29 min
          Check the https://jenkins.planx-pla.net/job/perf-sheepdog/job/PR-262/5/display/redirect

File: deployment/nginx/uwsgi.conf [hunk 1]
```diff
@@ -17,12 +17,13 @@ server {
         uwsgi_param REMOTE_ADDR $http_x_forwarded_for if_not_empty;
         uwsgi_param REMOTE_USER $http_x_userid if_not_empty;
         uwsgi_param GEN3_REQUEST_TIMESTAMP $msec;
-        uwsgi_param GEN3_TIMEOUT_SECONDS '45';
+        uwsgi_param GEN3_TIMEOUT_SECONDS $arg_timeout;
```
Comment (philloooo) [inline deployment/nginx/uwsgi.conf]: where is this set?

Comment (philloooo) [reply deployment/nginx/uwsgi.conf]: oh ok

**Comment by PlanXCyborg** [comment]
Jenkins Build 6 : time taken 28 min
          Check the https://jenkins.planx-pla.net/job/perf-sheepdog/job/PR-262/6/display/redirect

File: deployment/nginx/uwsgi.conf (hunk 1 above)
Comment (fantix) [reply deployment/nginx/uwsgi.conf]: :gratitude: Yes, thanks! Just for the record, it is set in Kubernetes configuration in the URL `/_status?timeout=2`.

**Pull request merged**